sql/core/src/main/scala/org/apache/spark/sql/SQLImplicits.scala

```diff
@@ -99,33 +96,96 @@ abstract class SQLImplicits {
 
   // Seqs
 
-  /** @since 1.6.1 */
-  implicit def newIntSeqEncoder: Encoder[Seq[Int]] = ExpressionEncoder()
+  /**
+   * @since 1.6.1
+   * @deprecated use [[newIntSequenceEncoder]]
+   */
+  def newIntSeqEncoder: Encoder[Seq[Int]] = ExpressionEncoder()
```

Comment:
oh you doesn't need to do this, just update the `project/MimaExcludes.scala` to add something like `ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.sql.SQLImplicits.newDoubleSeqEncoder")`. Please see the history of the `MimaExcludes.scala` to see how others update this file.